modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationEndpoints/CreateAnAPIApplicationEndpoint.testcase

```diff
@@ -58,16 +58,33 @@ definition {
 		}
 	}
 
-	@priority = 5
-	test CanCreateEndpointWithSiteScope {
+	@priority = 4
+	test CanCreateEndpointWithPathAndDescriptionWithSuccessAlert {
```

Comment:
Could you help double confirm in the jira ticket of the priority, and if it is 3, we should remove the `portal.acceptance`, thanks